### PR TITLE
TRIVIAL: Update gooddata-js changelog

### PR DESCRIPTION
**Pull request by ptquang86**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/171/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/202/ : SUCCESS in 1m 39s

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,15 +2,22 @@
...
-The REST API versions in the table are just for your information as the values are set internally and cannot be overridden. 
+The REST API versions in the table are just for your information as the values are set internally and cannot be overridden.
 
 |gooddata-js version | REST API version
 |:---:|:---:
 |\>= 10.0.0|3
 |<= 9.0.1|2
 
+<a name="11.1.0"></a>
+## 2019-01-22 Version [11.0.0](https://github.com/gooddata/gooddata-js/compare/v11.0.0...v11.1.0)
```
Comment (martinmosko) [inline CHANGELOG.md]: should be `[11.1.0]` :)

Comment (ptquang86) [reply CHANGELOG.md]: my bad :(

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/173/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/204/ : SUCCESS in 35s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/174/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/205/ : SUCCESS in 43s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/51/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/206/ : SUCCESS in 33s

**Pull request merged**